examples/rbpf_plugin.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright Microsoft Corporation
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+// Path: examples/rbpf_plugin.rs
+use std::io::Read;
+
+// Helper function used by https://github.com/Alan-Jowett/bpf_conformance/blob/main/tests/call_unwind_fail.data
+fn _unwind(a: u64, _b: u64, _c: u64, _d: u64, _e: u64) -> u64
+{
+    return a;
+}
+
+// This is a plugin for the bpf_conformance test suite (https://github.com/Alan-Jowett/bpf_conformance)
+// It accepts a single argument, the memory contents to pass to the VM.
+// It reads the program from stdin.
+fn main() {
+    let mut args: Vec<String> = std::env::args().collect();
+    let mut jit : bool = false;
+    let mut program_text = String::new();
+    let mut memory_text = String::new();
+
+    args.remove(0);
+    
+    // Memory is always the first argument.
+    if args.len() > 0 {
+        memory_text = args[0].clone();
+        // Strip whitespace
+        memory_text.retain(|c| !c.is_whitespace());
+        args.remove(0);
+    }
+
+    // Process the rest of the arguments.
+    while args.len() > 0 {
+        match args[0].as_str() {
+            "--help" => { 
+                println!("Usage: rbpf_plugin [memory] < program");
+                return;
+            },
+            "--jit" => { jit = true; },
```

Comment:
Can we please error out if running on Windows and selecting the JIT? It doesn't work on Windows at this time :/. This should fix the Appveyor builds.

Probably something like this:

```suggestion
            "--jit" => {
                #[cfg(windows)] {
                    println!(...);
                    return;
                }
                jit = true;
            },
```

Reply:
Looks like a bug somewhere in bpf_conformance. The "--cpu_version v2"  option is supposed to match what LLVM will produce when you specify v2 of the BPF CPU, but for some reason, it things one of the instructions in that test is only part of the v3 mode.

Reply:
Double checked. The only tests being skipped are:
1) 32bit jumps (EBPF_CLS_JMP32 op class)
2) Atomic store operations (EBPF_CLS_MASK op class + atomic flag).

Reply:
If you prefer, there is also an option to specify which tests to skip via regex (rather than the LLVM's concept of BPF CPU version). 
1) --cpu_version v2
2) --exclude_regex "(j.*32|lock)"

Reply:
It would probably be a good idea. This would allow us to run the conformance tests and to actually block if anything breaks. It will hide the results for the tests that are currently not working, but at the same time, these tests won't start working all of a sudden until we actually add support for the missing instructions in rbpf, so that's not really an issue.

Reply:
OK, tests updated and filed issues for all cases that don't pass that should pass (or pass on Linux at least).